modules/samples-operator-configuration.adoc

```diff
@@ -70,33 +83,50 @@ The Samples Operator still processes secrets while in `Removed` state. You can
 create the secret before switching to `Removed`, while in `Removed` before
 switching to `Managed`, or after switching to `Managed` state (though there are
 delays in creating the samples until the secret event is processed if you create
-the secret after switching to `Managed`). This is done to help facilitate the
+the secret after switching to `Managed`). This helps facilitate the
 changing of the registry, where you choose to remove all the samples before
 switching to insure a clean slate (removing before switching is not required).
 
 == Conditions
 
 The samples resource maintains the following conditions in its status:
 
-* *SamplesExists.* Indicates the samples are created in the OpenShift
-namespace.
-* *ImageChangesInProgress.*
-** `True` when imagestreams are created/updated, but
+[cols="3a,8a",options="header"]
+|===
+|Condtion |Description
+
+|`SamplesExists`
+|Indicates the samples are created in the OpenShift namespace.
+
+|`ImageChangesInProgress`
+|`True` when imagestreams are created/updated, but
 not all of the tag spec generations and tag status generations match.
-** `False` when all of the generations match, or unrecoverable errors occurred during
+
+`False` when all of the generations match, or unrecoverable errors occurred during
 import, the last seen error is in the message field. The list of pending
 imagestreams is in the reason field.
-* *ImportCredentialsExist.* A 'samples-registry-credentials' secret is
-copied into the OpenShift namespace.
-* *ConfigurationValid.* `True` or `false` based on whether any of the restricted
-    changes noted previously are submitted.
-* *RemovePending.* Indicator that there is a`Management State` `Removed` setting
-    pending, but are waiting for in progress imagestreams to complete.
-* *ImportImageErrorsExist.* Indicator of which imagestreams had errors during
-    the image import phase for one of their tags.
-** `True` when an error has occurred. The list of imagestreams with an error is
+
+|`ImportCredentialsExist`
+|A `samples-registry-credentials` secret is copied into the OpenShift namespace.
+
+|`ConfigurationValid`
+|`True` or `false` based on whether any of the restricted changes noted
```

Comment:
Capitalization is different between True and false. I assume it's case sensitive and they should be the same?